from unittest import TestCase


class TestSolution(TestCase):
    def test_bf_two_sum(self):
        #self.fail()
        pass
